src/utils/results-reporter.ts

```diff
@@ -98,7 +97,7 @@ export class ResultsReporter {
       if (totalScreens > 0) {
         await this.setStatusComment({
           createIfDoesNotExist: true,
-          body: `✅ ${METICULOUS_MARKDOWN_LINK} spotted zero visual differences across ${totalScreens} screens tested. (commit: ${this.shortHeadSha})`,
+          body: `✅ Meticulous spotted zero visual differences across ${totalScreens} screens tested: [view results](${testRun.url}).`,
```

Comment:
Added the link now we have a better screen for success results.

Reply:
When `No diffs summary page` PR merges:

```suggestion
          body: `✅ Meticulous spotted zero visual differences across [${totalScreens} screens tested](${testRun.url}/overview): [view results](${testRun.url}).`,
```

Reply:
We can also link to `${testRun.url}` directly even on no-diff runs, to keep it as the canonical way of linking to a test run for simplicity. Client-side we can redirect/replace with `/overview` in the future if it provides better UX than the current one.